### DISCUSSION TITLE
Can this library support spacy 3.6.0?

### DISCUSSION THREAD

**Discussion by Yihtsy** [HLasse/TextDescriptives#258] [General]
It is kind of inconvenient if I have to oscillate between spacy 3.5.4 and 3.6.0. Hopefully it would support the latest version of SpaCy in the future.

**Comment by KennethEnevoldsen** [comment]
I actually think it does, we haven't updated the upper bounds on versions. I have created PR #259. If the checks passed it will be merged in (it they don't I will update the package such that it support 3.6). Thanks for letting us know.

**Comment by HLasse** [comment]
Thanks both. Tests passed and #259 has been merged. @Yihtsy do let us know if you run into any issues using SpaCy 3.6!

**Reply by Yihtsy** [reply]
I think it works. Thank you very much.
